system/jlib/jtrace.cpp

```diff
@@ -165,7 +181,7 @@ class JLogSpanExporter final : public opentelemetry::sdk::trace::SpanExporter
     {
         if (isShutDown())
         {
-            ERRLOG("JLog Trace Exporter: Export failed, exporter is shutdown");
+            //Not logging this event per code review comments
```

Comment:
This isn't a useful comment in the code- it would be fine as a comment on the PR.  It begs the question what were the code review comments!  If you disagree, or don't understand the reason it should be discussed in the PR.